languages/ruby/exercises/concept/instance-variables/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+## Object state, instance variables
+
+Objects can hold their own state by setting _instance variables_. _Instance variables_ are created by prefixing `@` to the variable name.
+
+```ruby
+@instance_variable = 2
```

Comment:
I suggest not actually calling this `instance_variable` in case people misconstrue that as a formal part of the naming.

```suggestion
@name = 2
```